main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -542,8 +555,8 @@ object Weeder extends Phase[ParsedAst.Program, WeededAst.Program] {
       //
       // let* x = exp1; exp2     ==>   flatMap(x -> exp2)(exp1)
       //
-      val qname = Name.mkQName(Name.Ident(SourcePosition.Unknown, "flatMap", SourcePosition.Unknown))
-      val flatMap = WeededAst.Expression.VarOrDefOrSig(qname, loc)
+      val ident = Name.Ident(SourcePosition.Unknown, "flatMap", SourcePosition.Unknown)
```

Comment:
Just because you are already editing here, can I ask you to use sp1 and sp2 instead of SP.Unknown?

I want to slowly, but methodically get rid of all these Unknowns.

In fact a colleague had a problem with let* the other day and the source location was missing!